src/components/Main/SkeletonLoading.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+export const Skeleton = () => {
```

Comment:
Maybe this file should be TVPosterSkeleton. and changes references as well. We might have more skeletons in the future

Reply:
sounds good - I will change the naming convention